doc/changelog/1.22.0-changelog.rst

```diff
@@ -23,7 +23,7 @@ names contributed a patch for the first time.
 * Anirudh Dagar +
 * Ankit Dwivedi +
 * Antony Lee
-* Arfy Slowy +
+* Arfy Slowly +
```

Comment:
Unless you've contacted the author (unlikely?)--probably not desirable to assume that their surname should be "spell checked."

Reply:
Fixed. Sorry.